src/App.css

```diff
@@ -4,15 +4,91 @@
   box-sizing: border-box;
   margin: 0;
   padding: 0;
+  font-size: large;
+  color: rgb(52, 46, 46);
+}
+
+/* header and main position on every page */
+main {
+  margin: 30px;
+}
+
+header {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+}
+
+nav {
+  display: flex;
```

Comment:
### Optional
- [ ] It would be better if we have an underline to indicate which route is active :+1:

Reply:
It's a very good suggestion, thank you🙂 I did it. Looks nicer.